### ISSUE TITLE
Left right navigation in compact node does not read the names of sub-nodes

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#85039]
Explorer, have a compact node a/b/c 
Turn on VoiceOver.
Navigate between the sub nodes via left / right arrow. Notice how the names of the nodes is not read out by the screen reader.
Maybe the subnodes should simpy have a `aria-selected` and aria-label set on them. I will experiment.

fyi @joaomoreno

**Comment by isidorn** [comment]
The issue here is a bit more complex.
The screen reader properly reads out the `aria-label` of tree items because how the tree sets the `aria-activedescendent`.
So the explorer should somehow be able to dynamicaly set the `aria-activedescendent` of the tree. So that the activedescendent can be one of the subnodes.
@joaomoreno let me know what you think

**Comment by joaomoreno** [comment]
Ufff... maybe we need some tree infrastructure for this.

**Comment by isidorn** [comment]
Yeah we do. I would need a call on the tree to set the `activedescendent` to what I want.
I would generate ids on the subnodes, and based on subnode focus change the `activedescendent` on the tree

**Comment by isidorn** [comment]
Even if you expose dynamic setting of `activedescendent` the issue is that I do not have a reference to the tree from the controller, so I would have to wire that in somehow.
`IAccessibilityProvider` could have something like `onDidChangeActiveDescendant` which would fire with an string id.

**Comment by alexr00** [comment]
Using NVDA to test: clicking on any of the compact items expands and collapses the node. However, it  doesn't read "Expanded" and "Collapsed" the way non-compact nodes do. https://github.com/microsoft/vscode/issues/86414
